FannyAndersson/PayWay
# Feature/test update user

Test update user route. Route is 'handmade' and checks if user with req.params.id exists in db and is logged in (session.user) and return updated user data.

UPD:

Test it in Postman:
1. Log in as Muffin user;
2. test /api/profile/:id with some updated info - the best to change is name or activated. Response should contain user object.
3. logout and log in as another user
4. try to update user Muffin again - response should return error "Another user logged in"
5. Log in as admin (Muffin)
6.Try to update user Nemo. Response should contain updated Nemo, because user with role admin has access to update all users' info.
